src/coreclr/jit/hwintrinsiccodegenarm64.cpp

```diff
@@ -806,7 +808,8 @@ void CodeGen::genHWIntrinsic(GenTreeHWIntrinsic* node)
                     {
                         assert(intrin.baseType == TYP_DOUBLE);
                         assert(GetEmitter()->IsMovInstruction(ins));
-                        GetEmitter()->emitIns_Mov(ins, emitSize, targetReg, op1Reg, /* canSkip */ false, opt);
+                        bool canSkip = ((targetReg == op1Reg) && (intrin.baseType == intrin.op1->gtType));
```

Comment:
Same here, this one should always be skippable since `DuplicateToVector64(double)` is a "nop" (due to `double` being 64-bits and already in a SIMD register).

Reply:
>It would be good to confirm what path can hit this, however. Is it a synthesized node in lowering or elsewhere?

This was the smallest example I found in spmi replay:
[dump-225213.txt](https://github.com/dotnet/runtime/files/9533532/dump-225213.txt)

(The Assert hit was just added to catch this)